### ISSUE TITLE
Manjaro Gnome-Shell 3.22.1 not supported

### ISSUE THREAD

**Issue by joaoherberto** [eonpatapon/gnome-shell-extensions-mediaplayer#277]
Manjaro Gnome cannot upgrade to to Gnome 3.22.1 because mediaplayer extension requires < 3.21. Have to uninstall extension to ugrade Manjaro.

**Comment by aitbw** [comment]
Can confirm the extension doesn't work under Manjaro Gnome 3.22.1. Had to uninstall it as @joaoherberto did to upgrade my packages.

**Comment by dffischer** [comment]
I maintain [the Arch Linux User Repository package for this extension](https://aur.archlinux.org/packages/gnome-shell-extension-mediaplayer-git/), which is probably the one Manjaro users also install. I usually follow the compatibility declaration from [the extension metadata](https://github.com/eonpatapon/gnome-shell-extensions-mediaplayer/blob/master/src/metadata.json.in), which currently does not contain gnome-shell 3.22.

But I tested the extension on version 3.22 over the last few days and it seems to work fine, so I filed [a pull request to declare compatibility](https://github.com/eonpatapon/gnome-shell-extensions-mediaplayer/pull/279). Merging it should solve this problem.

Meanwhile, eager users may want to install ignoring dependencies (this would be `pacman -Ud *mediaplayer*.pkg.*"` and [disable extension version checks](https://www.maketecheasier.com/disable-extension-version-checks-gnome/).
